### DISCUSSION TITLE
image rotation

### DISCUSSION THREAD

**Discussion by 111lisu** [jomjol/AI-on-the-edge-device#801] [General]
I have a question regarding image rotation. in Edit Digits you can see normal, while in Overview it is twisted, but it was even more twisted earlier
![1.](http://server888056.nazwa.pl/alg_roi.jpg)
![1.](http://server888056.nazwa.pl/pobrane.png)

And now it gets worse again
![1.](http://server888056.nazwa.pl/alg_roi3.jpg)

**Comment by jomjol** [comment]
That happens, if your reference are not found properly. In this case the autoalignment algo fails with some arbitrary rotation angle.
You need to choose reference marks which are unique and have a high contrast.
